catalyst-gateway/bin/src/service/common/auth/rbac/token.rs

```diff
@@ -105,12 +106,14 @@ impl CatalystRBACTokenV1 {
         if catalyst_id.nonce().is_none() {
             return Err(anyhow!("Catalyst ID must have nonce"));
         }
-        let (role, rotation) = catalyst_id.role_and_rotation();
```

Comment:
Why removing these checks ?

Reply:
the previous check `if role != RoleIndex::ROLE_0 {` and `if rotation != KeyRotation::DEFAULT` doesn't work because there is no role and rotation ein auth data, this result in default value role = 0 and rotation = 0

So adding regex that check for last "/" follow by number would solve the problem,